src/events/crawler/scrapers/AUS/VIC/index.js

```diff
@@ -22,7 +22,7 @@ const scraper = {
     const currentArticleUrl = $anchor.attr('href');
     const $currentArticlePage = await fetch.page(`https://www.dhhs.vic.gov.au${currentArticleUrl}`);
     const paragraph = $currentArticlePage('.page-content p:first-of-type').text();
-    const { casesString } = paragraph.match(/cases in Victoria to (?<casesString>\d+)./).groups;
+    const { casesString } = paragraph.match(/cases in Victoria .* (?<casesString>\d+)./).groups;
```

Comment:
```suggestion
    const { casesString } = paragraph.match(/cases in Victoria \w* (?<casesString>\d+)./).groups;
```

I found `.*` to skip the case count and go to the next number (55)

Reply:
Even better. Cheers Q